Lean4Axiomatic/Natural/Exponentiation.lean

```diff
@@ -362,12 +364,12 @@ multiplication, then that's just the original number.
 
 **Proof intuition**: Exapansion of definitions and simplification.
 -/
-theorem pow_one {x : α} [AA.Identity (1:α) (· * ·)] : x^1 ≃ x := calc
-  _ = x^1        := rfl
-  _ ≃ x^(step 0) := pow_substR literal_step
-  _ ≃ x^0 * x    := pow_step
-  _ ≃ 1 * x      := AA.substL pow_zero
-  _ ≃ x          := AA.identL
+theorem pow_one {x : α} : x^1 ≃ x := calc
+  _ = x^1         := rfl
+  _ ≃ x^(step 0)  := pow_substR literal_step
+  _ ≃ x^(0) * x := pow_step
+  _ ≃ 1 * x       := AA.substL pow_zero
+  _ ≃ x           := identL
```

Comment:
Nitpick: align all of the `:=` operators. And I think the parens aren't needed around that `(0)`?